### PR TITLE
Some `nix-shell`-related suggestions

### PR DESCRIPTION
**Pull request by toraritte**
Thank you for making this (and all your other great projects and resources)!

### CODE REVIEW

File: ci.nix [hunk 1]
```diff
@@ -1,15 +1,28 @@
...
-    inherit (pkgs) gnumake which;
-
-    inherit (default) purs pulp purp psc-package dhall-simple spago pscid spago2nix purty zephyr;
-  };
+  inputsFromPkgs = with pkgs; [
+    gnumake
+    which
+  ];
 
+  default = import ./default.nix {};
```
Comment (justinwoo) [inline ci.nix:9]: please, i do not want to use this method

File: README.md [hunk 2]
```diff
@@ -4,7 +4,13 @@ A project for using PureScript and related tooling easily with Nix. Note that th
 
 ## Example usage
 
-See [ci.nix](./ci.nix) in this repo for example `nix-shell` usage.
+See [ci.nix](./ci.nix) in this repo for a Nix expression example to be used with `nix-shell`.
```
Comment (justinwoo) [inline README.md:7]: this is fine

File: ci.nix (hunk 1 above)
Comment (toraritte) [reply ci.nix:9]: So use the commented out line below? (`inputsFromDefault = with (import ./default.nix {}); [`) Sorry, not sure what you had in mind.

Comment (justinwoo) [reply ci.nix:9]: i do not want to use the changes in this file, and i especially do not ever want `with` in my code

Comment (toraritte) [reply ci.nix]: Removed. (Posting this here as a reminder to myself: [Nix anti-patterns: `with attrset; ...` expression](https://nix.dev/anti-patterns/language.html#with-attrset-expression))

**Pull request merged**